php/utils.php

```diff
@@ -511,14 +511,15 @@ function mustache_render( $template_name, $data = array() ) {
  *
  * @param string  $message  Text to display before the progress bar.
  * @param integer $count    Total number of ticks to be performed.
+ * @param int     $interval  The interval in milliseconds between updates.
```

Comment:
Extra space before the "The". Also could you expand the comment to
```
Optional. The interval in milliseconds between updates. Default 100.
```
?

Reply:
I copy/pasted the doc from the implementing class, would you like wording updated there as well for consistency?

Reply:
Well, it'd be nice, but probably not worth it!

Reply:
Yeah, the class is upstream in PHP CLI tools so not worth it. :)